.dockerignore

```diff
@@ -1,8 +0,0 @@
-*
```

Comment:
This file was left behind by a previous PR. Originally, this repo had a Dockerfile (deleted) that was used to run tests on Linux. But we needed tests to run on Windows and macOS too, and we don't have a "Windows containers" setup, so I configured AppVeyor to run tests with `npm test` outside any container.